examples/getTimeSecs-demo/index.mjs

```diff
@@ -0,0 +1,19 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+const stdlib = loadStdlib(process.env);
+
+const startTime = await stdlib.getNetworkTime();
+const accAlice = await stdlib.newTestAccount(stdlib.parseCurrency(100));
+const ctcAlice = accAlice.contract(backend);
+await ctcAlice.p.Alice({
+  giveTime: async (time) => {
+    console.log(`The contract was launched at network time: ${time}`);
+    console.log(`which is network seconds: ${await stdlib.getTimeSecs(time)}`);
+  }    
+});
+console.log(`The program started at network time ${startTime}`);
+console.log(`which is network seconds: ${await stdlib.getTimeSecs(startTime)}`);
+await stdlib.wait(100);
```

Comment:
Eh... I am actually about to make a rule that we almost never call `wait` in tests, because it causes random failures with other tests. This is going to force the network forward 100 blocks which is going to mess up timeouts for other people. I think it is okay to just delete 16-19

Reply:
Got it.